Unable to resolve dependencies from artifactory requiring authorization

**Describe the bug**


```
15:27:06: Executing task 'refreshVersions'...

<-- 401 Unauthorized https://custom-repo.com/.../maven-metadata.xml (231ms, 91-byte body)

> Task :refreshVersions FAILED
1 actionable task: 1 executed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':refreshVersions'.
> ModuleId(group=de.xx.yyy, name=zzz) not found. Searched the following repositories:
  - file:/Users/pse/.m2/repository/
  - https://repo.spring.io/milestone/
  - https://repo.maven.apache.org/maven2/
  - https://jcenter.bintray.com/
  - https://custom-repo.com/

```

build.gradle:

```
repositories {
    mavenLocal()
    maven { url = uri("https://repo.spring.io/milestone") }

    mavenCentral()
    jcenter()
    maven {

        val artifactory_username: String by project
        val artifactory_password: String by project

        url = uri("https://custom-repo.com")
        credentials {
            username = artifactory_username
            password = artifactory_password
        }
    }
}
```

repo.gradle:

```
project.ext {
    artifactoryConfiguration = {
        url "https://custom-repo-com"
        credentials {
            username artifactory_username // defined in gradle.properties
            password artifactory_password // defined in gradle.properties
        }
    }
}

```

**Additional context**
Additional context

Gradle Version: 6.5.1
Kotlin Version: 1.4.10
RefreshVersions Version: 0.9.7 

**When downgrading to version 0.9.5 the build succeeds.**

--- a/plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/extensions/gradle/AuthenticationSupported.kt
+++ b/plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/extensions/gradle/AuthenticationSupported.kt
@@ -9,5 +9,5 @@ val AuthenticationSupported.passwordCredentials: PasswordCredentials?
     // https://github.com/gradle/gradle/issues/14694
     get() = runCatching {
         // We use runCatching to avoid crashing the build if the internal APIs change.
-        (this as AuthenticationSupportedInternal?)?.configuredCredentials as? PasswordCredentials
+        (this as AuthenticationSupportedInternal?)?.configuredCredentials?.orNull as? PasswordCredentials
     }.getOrNull()

